src/pages/MyPage/MyPage.style.js

```diff
@@ -196,6 +196,9 @@ const BoxWrapper = styled.div`
 `;
 
 const ButtonWrapper = styled.div`
+  width: 100%;
+  position: relative;
+  text-align: -webkit-center;
```

Comment:
-webkit-center을 쓰신 이유가 있을까요?

Reply:
해당 래퍼는 저장이라는 버튼과 '회원 탈퇴 >' 를 묶어놓은 래퍼입니다.

회원 탈퇴 텍스트를 absolute 로 설정하고 컨테이너를 width 100% 으로 하고 버튼을 중앙으로 하려고 -webkit-center 를 사용했습니다.
그냥 center 는 안되고.. -webkit-center 만 동작하더라구요.

혹시 좋은 방법이 있을까요?

Reply:
```suggestion
  display: flex;
  justify-content: center;
```

flexbox로 처리하면 좋을 것 같습니다!

Reply:
오 감사합니다. 리뷰 적용했습니다